wcivf/apps/parties/management/commands/import_parties.py

```diff
@@ -16,8 +17,31 @@ def handle(self, **options):
             self.add_parties(results)
             next_page = results.get("next")
 
+    def add_party_descriptions(self, party_obj, descriptions):
+        for description in descriptions:
+            try:
+                party_obj.party_descriptions.get_or_create(
+                    party=party_obj,
+                    description=description["description"],
+                    date_description_approved=description[
+                        "date_description_approved"
+                    ],
+                    defaults={
+                        "date_description_approved": description[
+                            "date_description_approved"
+                        ]
+                    },
```

Comment:
This is a mix of a few things we talked about. I think we should use `update_or_create` here, passing in `party=` and `description=` as kwargs but _not_ `date_description_approved`. That can stay in the `defaults` dict as it is at the moment.

This, translated means: 

1. Find a model with the same party and description as given (by the kwargs)
2. If it exists, update the model to set the  `date_description_approved`
3. If not, make the model with all the kwargs and defaults.